src/transaction_builder.js

```diff
@@ -367,6 +384,9 @@ TransactionBuilder.prototype.__build = function (allowIncomplete) {
   if (!allowIncomplete) {
     if (!this.tx.ins.length) throw new Error('Transaction has no inputs')
     if (!this.tx.outs.length) throw new Error('Transaction has no outputs')
+
+    // do not rely on this, its merely a last resort
+    if (this.__hasAbsurdFee()) throw new Error('Transaction has absurd fees')
```

Comment:
The fee wouldn't be absurd if the transaction is very large, or you really want it to confirm. It feels it should be possible without setting `allowIncomplete = true`